RF_WRK/README.md

```diff
@@ -0,0 +1,11 @@
+# Richard Version
+
+*I wasn't sure how to add this to the inital repo, so I put my stuff in this folder so as not to mess up however your stuff is organized.*
```

Comment:
I think this is the most important thing we should determine before continuing with this review. From your perspective, is this an addition/enhancement of the original project or a new project?

I am asking, since if it is the latter then it deserves to be in its own repository. Otherwise we'll need to do some changes in the structure and documentation in order to make it consistent with the rest. Let me know what you think so we can move this forward! :+1:

Reply:
Just checked out the schematics. Looks very cool, nice idea and execution! :smiley:

However, I think that this is a _unique_ project that can and can and should stand on its own as it follows a different concept (with multiple boards), Arduino footprint, processing sketch that seems to be serving a specific purpose and so on.

This is a fascinating project and I'd feel bad for overshadowing it by stuffing it in a folder of this repo. :) Please let me know your thoughts on this.